### PR TITLE
Werewolves enjoy dog biscuits too.

### PR DESCRIPTION
**Pull request by Munien7**
## About the PR 

Werewolves can eat dog biscuits without getting the "That tasted awful! Why would you eat it!?" message, instead they get the same message as pugs, just without the healing buff.
werewolf

## Why's this needed? 

Big dogs deserve to enjoy treats just as much as little ones.

## Changelog

```changelog
(u)Munien
(+)Werewolves can now happily enjoy dog treats!
```

### CODE REVIEW

**Comment by ZeWaka** [comment]
Nah they should get the heal

File: code/modules/food_and_drink/snacks.dm [hunk 1]
```diff
@@ -412,6 +412,10 @@
 			if (ispug(M))
 				..()
 				boutput(M, "<span class='notice'>That tasted delicious!</span>")
+			if (iswerewolf(M))
+				..()
+				boutput(M, "<span class='notice'>That tasted delicious!</span>")
+
```
Comment (ZeWaka) [inline code/modules/food_and_drink/snacks.dm:418]: Gross, duplicate code

File: code/modules/food_and_drink/snacks.dm [hunk 2]
```diff
@@ -424,6 +428,10 @@
 				src.food_effects = list()
 			..()
 			src.food_effects = food_effects_pre
+			if (!iswerewolf(M))
```
Comment (ZeWaka) [inline code/modules/food_and_drink/snacks.dm:431]: I'd rather invert this

File: code/modules/food_and_drink/snacks.dm (hunk 1 above)
Comment (Munien7) [reply code/modules/food_and_drink/snacks.dm]: Still new to this sorry, realised how bad that reads

File: code/modules/food_and_drink/snacks.dm (hunk 2 above)
Comment (Munien7) [reply code/modules/food_and_drink/snacks.dm]: fixed poorly written duplicate, put it in with the previous condition with pugs

**Pull request merged**